gen_cert.py

```diff
@@ -173,6 +173,13 @@ def interstitial_factory():
         # pdfs differently for the different templates
         self.template_version = cert_data.get('VERSION', 1)
         self.template_type = 'honor'
+        # Default Font
+        self.template_font_name = 'OpenSans'
+        self.template_font_type = 'Light'
+        # Check for yaml template font
+        if cert_data.get('font'):
+            self.template_font_name = cert_data.get('font')['name']
+            self.template_font_type = cert_data.get('font')['type']
```

Comment:
I don't love this implementation of this...

I'd prefer this:
```python
self.template_font = cert_data.get('font', {})
self.template_font_name = cert_data.get(self.template_font['name'], 'OpenSans')
self.template_font_type = cert_data.get(self.template_font['type'], 'Light')
```

also, yours would explode if you had:
``` yaml
THE/COURSE/TRIPLE:
  font:
    name: "SourceSans"
```